cli/cli/out/file_logger.go

```diff
@@ -1,53 +1,107 @@
 package out
 
 import (
+	"fmt"
 	"github.com/kurtosis-tech/kurtosis/cli/cli/helpers/host_machine_directories"
 	"github.com/kurtosis-tech/stacktrace"
 	"github.com/sirupsen/logrus"
 	"io/fs"
 	"os"
+	"path"
+	"strconv"
+	"strings"
 	"sync"
+	"time"
 )
 
-var once sync.Once
+var (
+	once                      sync.Once
+	kurtosisCliLogsRootFolder = ""
+)
+
+const (
+	fileNamePrefix              = "kurtosis-cli"
+	numberOfLogFilesForCommands = 7
+)
 
 // TODO: In commands like inspect we use out.PrintOutLn - will need to add this fileLogger
 //  to print commands' output as well.
 // fileLogger this logger will only log to a file
 var fileLogger *logrus.Logger
 var permission = fs.FileMode(0666)
 
-func GetFileLogger() (*logrus.Logger, error) {
-	var err error
-
+func GetFileLogger() *logrus.Logger {
 	if fileLogger == nil {
 		once.Do(func() {
-			err = setupFileLogger()
+			setupFileLogger()
 		})
 	}
+	return fileLogger
+}
 
+func RemoveLogFiles() {
+	files, err := os.ReadDir(kurtosisCliLogsRootFolder)
 	if err != nil {
-		return nil, stacktrace.Propagate(err, "Error occurred while getting the file logger")
+		logrus.Warnf("Error occurred while listing the files from the directory at - '%v' with error %+v", kurtosisCliLogsRootFolder, err)
+	}
+
+	numberOfFilesToBeDeleted := len(files) - numberOfLogFilesForCommands
+	logrus.Debugf("Removing old %v log file(s) from path %v", numberOfFilesToBeDeleted, kurtosisCliLogsRootFolder)
+	if len(files) > numberOfLogFilesForCommands {
+		for i := 0; i < numberOfFilesToBeDeleted; i++ {
+			fileName := files[i].Name()
+			logFilePath := path.Join(kurtosisCliLogsRootFolder, fileName)
+			if strings.HasPrefix(fileName, fileNamePrefix) {
```

Comment:
I think that this condition should be analyzed before this iteration in order to filter the files list, otherwise you could end up without deleting any file